src/style.css

```diff
@@ -1 +1,97 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: local('Avenir');
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: local('Avenir-Book');
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir, sans-serif;
+  src: local('Avenir-Heavy');
+  font-weight: 900;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.page__body {
+  margin-top: 20px;
+}
+
+.page__search-form {
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  margin-top: 20px;
+  padding: 0;
+}
+
+.page__search-form:nth-child(1) {
+  margin-top: 0;
+}
+
+.search {
+  box-sizing: border-box;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position-y: center;
+  border: 1px solid #e1e7ed;
+  border-radius: 4px;
+  font-family: Avenir, sans-serif;
+  box-shadow: 0 1px 8px 0 #3d4e611a;
+}
+
+.search::placeholder {
+  font-family: Avenir, serif;
```

Comment:
The `font-family` for the placeholder is set to 'Avenir, serif'. Consider using 'Avenir, sans-serif' to maintain consistency with the rest of the document.